api/managementpb/backup/backups.proto

```diff
@@ -47,4 +67,11 @@ service Backups {
       body: "*"
     };
   }
+  // RestoreBackup requests the backup restore.
+  rpc RestoreBackup(RestoreBackupRequest) returns (RestoreBackupResponse) {
+    option (google.api.http) = {
+      post: "/v1/management/backup/Backups/RestoreBackup"
```

Comment:
Only now mentioned, how about `Backups/Restore` and `Backups/Start`?

Reply:
could be, need the second opinion on this.

Reply:
@atymchuk ^, I think u had some opinion on this.

Reply:
Agree with @artemgavrilov, `Backups/Restore` is a better option.

Reply:
renamed